### ISSUE TITLE
broken: mixcloud.com

### ISSUE THREAD

**Issue by thpublic** [rg3/youtube-dl#4784]
~/youtube-dl http://www.mixcloud.com/johndigweed/transitions-with-john-digweed-live-from-maze-athens/ --verbose 
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['http://www.mixcloud.com/johndigweed/transitions-with-john-digweed-live-from-maze-athens/', '--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out None, pref UTF-8
[debug] youtube-dl version 2015.01.23.4
[debug] Python version 2.7.6 - Linux-3.13.0-24-generic-i686-with-LinuxMint-17-qiana
[debug] exe versions: none
[debug] Proxy map: {}
WARNING: unable to extract like count; please report this issue on http://yt-dl.org/bug
WARNING: unable to extract play count; please report this issue on http://yt-dl.org/bug
ERROR: Unable to extract upload date; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/mint/youtube-dl/youtube_dl/YoutubeDL.py", line 604, in extract_info
    ie_result = ie.extract(url)
  File "/home/mint/youtube-dl/youtube_dl/extractor/common.py", line 260, in extract
    return self._real_extract(url)
  File "/home/mint/youtube-dl/youtube_dl/extractor/mixcloud.py", line 96, in _real_extract
    webpage, 'upload date'))
  File "/home/mint/youtube-dl/youtube_dl/extractor/common.py", line 516, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract upload date;

**Comment by Fusl** [comment]
Same error here.
```
 _ youtube-dl --verbose http://www.mixcloud.com/livesetsofhardstyle/coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance/
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.mixcloud.com/livesetsofhardstyle/coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance/']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.25
[debug] Python version 2.7.6 - Linux-3.11.10-25-desktop-x86_64-with-SuSE-13.1-x86_64
[debug] exe versions: ffmpeg N-68389-g13c3a97, ffprobe N-68389-g13c3a97
[debug] Proxy map: {}
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Downloading webpage
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 1/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 2/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 3/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 4/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 5/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 6/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 7/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 8/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 9/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 10/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 11/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 12/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 13/31 ...
[mixcloud] livesetsofhardstyle-coone-exclusive-mix-2014-dancing-astronaut-the-sound-of-q-dance: Checking URL 14/31 ...
WARNING: unable to extract like count; please report this issue on http://yt-dl.org/bug
WARNING: unable to extract play count; please report this issue on http://yt-dl.org/bug
ERROR: Unable to extract upload date; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 613, in extract_info
    ie_result = ie.extract(url)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/common.py", line 266, in extract
    return self._real_extract(url)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/mixcloud.py", line 96, in _real_extract
    webpage, 'upload date'))
  File "/usr/bin/youtube-dl/youtube_dl/extractor/common.py", line 522, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract upload date; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by dstftw** [comment]
mixcloud will work again in the next version. Thanks for the report.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.02. See [our FAQ](https://yt-dl.org/update) if you need help updating.
